angular-with-directus/how to integrate directus with Angular.md

```diff
@@ -0,0 +1,396 @@
+Angular is a front-end framework that leverage TypeScript to create web applications. In this tutorial, you will use the framework to implement the front-end for the Directus headless CMS. You will implement a blog that loads blog posts dynamically and also serves global metadata.
+
+## Prerequisites
+- Some knowledge of TypeScript and [Angular](https://angular.io/)
+- A Directus project either locally or on [Directus cloud](https://directus.cloud/register)
+- [Node.js](https://nodejs.org/en/download) and a development environment of your choice
+- Install the Angular CLI - use the Angular [guide](https://angular.io/guide/setup-local) to achieve this. Note that Angular and TypeScript versions must be compatible. Since the SDK requires a minimum TypeScript version of 5.0, you need to use Angular version 17 for your project.
+
+## Initialize Project
+To create a new Angular project, use the following command.
+
+```bash
+ng new directus-with-angular
+```
+For the prompts, choose CSS for the stylesheet format and no to disable server side rendering and static site generation.
+
+```bash
```

Comment:
Remove this. You don't need to explain the options then show the options. Putting them in one code block is much more succinct.

Reply:
sure